peer/brontide.go

```diff
@@ -1560,6 +1560,18 @@ out:
 			case <-p.quit:
 				break out
 			}
+
+			// Send the Shutdown to the link. If ProcessCloseMsg
```

Comment:
See comment above, don't see why we need to send the shutdown, given we already have the `closeMsg` (sent above), and can add other state there if needed.